src/ClickatellClient.php

```diff
@@ -40,27 +41,14 @@ public function send(array $to, $message)
     {
         $to = collect($to)->toArray();
 
-        $response = $this->clickatell->sendMessage($to, $message);
-
-        $this->handleProviderResponses($response);
-    }
-
-    /**
-     * @param array $responses
-     * @throws CouldNotSendNotification
-     */
-    protected function handleProviderResponses(array $responses)
-    {
-        collect($responses)->each(function (stdClass $response) {
-            $errorCode = (int) $response->errorCode;
-
-            if ($errorCode != self::SUCCESSFUL_SEND) {
-                throw CouldNotSendNotification::serviceRespondedWithAnError(
-                    (string) $response->error,
-                    $errorCode
-                );
-            }
-        });
+        $client = new \GuzzleHttp\Client(['base_uri' => 'https://platform.clickatell.com/messages/http/']);
```

Comment:
I'm a bit confused here, we seem to be importing the client but using guzzle?

Reply:
This was a temporary workaround for an internal project. We'll be cleaning it up back to the Acturial library soon. Thanks for the review though!

Reply:
No worries :)